IronSoftware.Drawing/nuget.config

```diff
@@ -3,5 +3,6 @@
 	<packageSources>
 		<clear />
 		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+		<add key="Common" value="https://pkgs.dev.azure.com/ironcoders/IronCommon/_packaging/IronCommon/nuget/v3/index.json" />
```

Comment:
Does this cause issue to the customer endpoint when they need to use the package since they have no access to `https://pkgs.dev.azure.com/ironcoders/IronCommon/_packaging/IronCommon/nuget/v3/index.json`?

Reply:
We have `IronSoftware.Drawing.Abstractions` which will publish to nuget. But this using when we develop internal.